src/views/FindOrder.vue

```diff
@@ -28,57 +28,82 @@
           <ion-list>
             <ion-list-header>{{ $t("Date") }}</ion-list-header>
             <ion-item>
-              <ion-label>order created</ion-label>
-              <ion-select value="any">
-                <ion-select-option value="any">any</ion-select-option>
-              </ion-select>
+              <ion-label>{{ $t("Order created") }}</ion-label>
+              <ion-chip id="open-order-created-date-modal" slot="end">
```

Comment:
I think this is not action but an id
```suggestion
              <ion-chip id="order-created-date-modal" slot="end">
```

Reply:
Same applies to other ids

Reply:
Sure Sir, will update the ids